tests/test_client.py

```diff
@@ -1104,3 +1104,18 @@ def test_multiple_positional_args(sentry_init):
     with pytest.raises(TypeError) as exinfo:
         sentry_init(1, None)
     assert "Only single positional argument is expected" in str(exinfo.value)
+
+
+@pytest.mark.parametrize(
+    "sdk_options, expected_data_length",
+    [({}, DEFAULT_MAX_STRING_LENGTH), ({"max_string_length": 2000}, 2000)],
+)
+def test_max_string_length_option(
+    sentry_init, capture_events, sdk_options, expected_data_length
+):
+    sentry_init(sdk_options)
+    events = capture_events()
+
+    capture_message("a" * 2000)
```

Comment:
Should we set the length of the string to something bigger than 2000 to make it more clear that some trimming is happening?